core/templates/domain/utilities/audio-language.model.ts

```diff
@@ -35,19 +32,13 @@ export class AudioLanguage {
     this.description = description;
     this.relatedLanguages = relatedLanguages;
   }
-}
 
-@Injectable({
-  providedIn: 'root'
-})
-export class AudioLanguageObjectFactory {
-  createFromDict(audioLanguageDict: AudioLanguageBackendDict): AudioLanguage {
+  static createFromDict(
```

Comment:
Can you share the link to the file where this function is used? (I wanted to check why we are creating a frontendDict for this.)

Reply:
[language-util.service.ts Line 68](https://github.com/oppia/oppia/pull/10713/files#diff-569ba9a2f5e25eee34f133695b0d8149)